packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -317,7 +317,7 @@ export type IntervalStickiness = (typeof IntervalStickiness)[keyof typeof Interv
 export enum IntervalType {
     // (undocumented)
     Simple = 0,
-    SlideOnRemove = 2,
+    SlideOnRemove = 2,// SlideOnRemove is default behavior - all intervals are SlideOnRemove
```

Comment:
New typescript seems to result in preserved trailing comments in some cases like this. Interesting, but I think a non-issue.

Reply:
I seem to remember @DLehenbauer mentioning that we should try to upgrade api-extractor and typescript at the same time and I think these sorts of things was one reason. Though @Josmithr has also updated api-extractor independently so I could be way off here.

Reply:
We are already using a version of API extractor which uses newer typescript than we build with. This change brings them in alignment again (if our typescript is newer than theirs, we get warnings)